.pre-commit-config.yaml

```diff
@@ -30,27 +23,31 @@ repos:
         - file
         - python
 
-    - entry: isort
-      exclude: "^.*/js/.*$"
-      files: ""
-      id: isort
-      language: python
-      name: isort
-      pass_filenames: true
-      stages:
-        - commit
-      types:
-        - python
-
-    - entry: mypy
-      exclude: ^$
-      # note: it would be good to extend this to other apps as well
-      files: "{{ cookiecutter.package_slug }}/"
-      id: mypy
-      language: python
-      name: mypy
-      pass_filenames: true
-      stages:
-        - commit
-      types:
-        - python
+  - repo: local
```

Comment:
repo: local was already defined in line 7,
so you don't need to redeclare it .. neither hooks:

Reply:
Don't creat a new repo local here, instead, move ruff to the existent repo local after line 24